examples/backends/vllm/launch/elastic_ep.sh

```
@@ -0,0 +1,86 @@
+# have free GPUs. This is distinct from `--headless` (mp-backend multi-node).
+#
+# Scale at runtime (control surface is served on DYN_SYSTEM_PORT):
+#   curl -X POST localhost:${DYN_SYSTEM_PORT:-8081}/engine/control/scale_elastic_ep \
+#        -H 'Content-Type: application/json' \
+#        -d '{"new_data_parallel_size": 4}'
+
+set -e
+trap 'echo Cleaning up...; kill 0' EXIT
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/launch_utils.sh" # print_launch_banner, wait_any_exit
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)